launch/tier4_autoware_api_launch/launch/autoware_api.launch.xml

```diff
@@ -1,6 +1,6 @@
 <launch>
   <!-- AD API options -->
-  <arg name="launch_deprecated_api" default="false"/>
+  <arg name="launch_deprecated_api" default="true"/>
```

Comment:
```suggestion
  <arg name="launch_deprecated_api" default="false"/>
```

Reply:
Thanks. As I commented [here](https://github.com/autowarefoundation/autoware.universe/pull/3147#issuecomment-1480667392), I'll remove this diff.

Reply:
I've removed it.